CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## UNRELEASED
 
+* **Metrics**
+  * [FIX] - Restored various container-related metrics that were being filtered out after an update.
+
```

Comment:
You have an outdated version of CHANGELOG.md.  The move to FB 1.9.9 was included in our (unexpected) 1.2.7 release.